src/cljs/kefirnadar/application/styles.cljs

```diff
@@ -14,4 +14,74 @@
 (defn use-styletron
   []
   (let [[css] (styletron-react/useStyletron)]
-    [(comp css clj->js)]))
\ No newline at end of file
+    [(comp css clj->js)]))
+
+(def styles-map {:wrapper       {:max-width "90%"
```

Comment:
zanimljiva ideja da stavis sve stilove u jedan var. To je s jedne strane dobro jer nam stilovi ne smetaju u komponentama, ne zauzimaju puno mesta. Ali s druge strane je malo cimanje kada moras da menjas neki stil jer moras da se setas izmedju fajlova. Hajde neka ostane ovako za sad pa cemo videti kako ce se pokazati.

Reply:
Dogovoreno 😁👍👍
U sustini svi su u fajlu za stilove, meni nije bilo problem, ako bude trebalo da se menja menjam